tests/basic_integration/fns.sh

```
@@ -16,6 +16,7 @@ fi
 RPC_URL="ws://localhost:8545"
 
 PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+NETWORK_PRIVATE_KEY_AG="0x51a1e500a548b70d88184a1e042900c0ed6c57f8710bcc35dc8c85fa33d3f580"
```

Shellcheck (0.10.0)

[warning] NETWORK_PRIVATE_KEY_AG appears unused. Verify use (or export if used externally).

(SC2034)